newscoop/classes/Article.php

```diff
@@ -982,6 +989,37 @@ public function getSection()
 
 
     /**
+     * Check whether a copy with the same values as this object, but with one or more changed, will violate uniqueness constraints
+     *
+     * @param array $changedValues -
+     *                           Array of key value pairs to replace those of the this object
+     *
+     * @return boolean - true if the object with changed values will be unique
+     */
+    public function willReplacementBeUnique($changedValues)
```

Comment:
Yeah, sorry. I thought I'd removed that.

On Thu, Mar 31, 2016 at 6:03 PM, Paweł Mikołajczuk <notifications@github.com
> wrote:

> In newscoop/classes/Article.php
> <https://github.com/sourcefabric/Newscoop/pull/1351#discussion_r58080317>:
>
> > @@ -982,6 +989,37 @@ public function getSection()
> >
> >
> >      /**
> > +     * Check whether a copy with the same values as this object, but with one or more changed, will violate uniqueness constraints
> > +     *
> > +     * @param array $changedValues -
> > +     *                           Array of key value pairs to replace those of the this object
> > +     *
> > +     * @return boolean - true if the object with changed values will be unique
> > +     */
> > +    public function willReplacementBeUnique($changedValues)
>
> Hey, looks like this function is not used anywhere.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/sourcefabric/Newscoop/pull/1351/files/a2bdf5d81b1001eda0870f6231c26fb7024be4c5#r58080317>
>